apps/web/src/pages/Pools/ViewPoolPage/components/SubmitForPublishingDialog.tsx

```diff
@@ -0,0 +1,112 @@
+import React from "react";
+import { useIntl } from "react-intl";
+
+import { Button, Dialog, Link } from "@gc-digital-talent/ui";
+import { toast } from "@gc-digital-talent/toast";
+import { Maybe } from "@gc-digital-talent/graphql";
+
+const contactLink = (chunks: React.ReactNode) => (
+  <Link
+    color="secondary"
+    external
+    href="mailto:recruitmentIMIT-recrutementGITI@tbs-sct.gc.ca"
+  >
+    {chunks}
+  </Link>
+);
+
+interface SubmitForPublishingDialogProps {
+  isReadyToPublish: Maybe<boolean>;
+}
+
+const SubmitForPublishingDialog = ({
+  isReadyToPublish,
+}: SubmitForPublishingDialogProps) => {
+  const intl = useIntl();
+
+  const dialogTitle = intl.formatMessage({
+    defaultMessage: "Submit for publishing",
+    id: "I5q1n6",
+    description: "Title for dialog to submit a process for publishing",
+  });
+
+  if (!isReadyToPublish) {
+    return (
+      <Button
+        mode="inline"
+        color="primary"
+        onClick={() => {
+          toast.error(
+            intl.formatMessage({
+              defaultMessage:
+                "Please complete all advertisement information before publishing.",
+              id: "QzU2SL",
+              description:
+                "Error message displayed when user attempts to publish an incomplete advertisement",
+            }),
+          );
+        }}
+      >
+        {dialogTitle}
+      </Button>
+    );
+  }
+
+  return (
+    <Dialog.Root>
+      <Dialog.Trigger>
+        <Button mode="inline" color="primary">
+          {dialogTitle}
+        </Button>
+      </Dialog.Trigger>
+      <Dialog.Content>
+        <Dialog.Header>{dialogTitle}</Dialog.Header>
+        <Dialog.Body>
+          <p data-h2-margin-bottom="base(x1)">
+            {intl.formatMessage({
+              defaultMessage:
+                " Your advertisement information and assessment plan are complete.",
+              id: "SQJgGP",
+              description:
+                "Lead-in text on a complete process, ready for publishing",
+            })}
+          </p>
+          <p data-h2-margin="base(x1 0)">
+            {intl.formatMessage(
+              {
+                defaultMessage:
+                  "If you are ready to publish this advertisement please <a>get in touch with an administrator</a>.",
+                id: "EqUTLs",
+                description:
+                  "Instructions on how to submit a process for publishing",
+              },
+              { a: contactLink },
+            )}
+          </p>
+          <p data-h2-margin="base(x1 0)">
+            {intl.formatMessage({
+              defaultMessage:
+                "The GC Digital talent team will review and publish your advertisement.",
```

Comment:
I am pretty sure we are still GC Digital Talent, no? Don't forget to change the id!
```suggestion
                "The GC Digital Talent team will review and publish your advertisement.",
```